common/app/model/content.scala

```diff
@@ -304,6 +306,17 @@ final case class Content(
   val nonCompliantOutbrainAmp = (hasStoryPackage && tags.series.nonEmpty) || (tags.series.length > 1)
 }
 
+object CardStylePicker {
+  def apply(content: contentapi.Content, trailMetaData: MetaDataCommonFields): com.gu.facia.api.utils.CardStyle = {
```

Comment:
and if there is a naming clash here or elsewhere (e.g. SpecialReport) just use an alias in the import?